framework/tlvf/tlvf_conf.yaml

```diff
@@ -18,6 +19,8 @@ include_yaml_path: {
   "tlvf/wfa_map/tlvRadioOperationRestriction.yaml",
   "tlvf/wfa_map/tlvChannelSelectionResponse.yaml",
   "tlvf/wfa_map/tlvTransmitPowerLimit.yaml",
+  "tlvf/wfa_map/tlvSteeringBTMReport.yaml",
```

Comment:
This should definitely be a separate commit. Also, the AutoGenerated seems to be missing.